csrc/serde/python_fusion_cache_generated.h

```diff
@@ -206,51 +206,52 @@ enum RecordType {
   RecordType_IotaOp = 14,
   RecordType_IndexSelectOp = 15,
   RecordType_TorchGatherOp = 16,
-  RecordType_Unary_TV = 17,
```

Comment:
not really for this PR. @rdspring1 are we versioning serialized models? I imaging these changes would render them incompatible across builds.

Reply:
Does this mean we will need to bump that version for changes like this?

Reply:
I don't think so because there aren't any users saving kernels for reuse yet.